mango-logs/src/lib.rs

```diff
@@ -1,15 +1,59 @@
 use anchor_lang::prelude::*;
+use anchor_lang::Discriminator;
+use std::io::Write;
 declare_id!("Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS");
 
-// Log to Program Log with a prologue so transaction scraper knows following line is valid mango log
+/// Log to Program Log with a prologue so transaction scraper knows following line is valid mango log
+///
+/// Warning: This will allocate large buffers on the heap which will never be released. Since the max
```

Comment:
nitpick: could you please add a small comment why will never be released for future readers unfamiliar with solana runtime